index.js

```diff
@@ -28,14 +31,21 @@ AFRAME.registerComponent('cubemap', {
 
     // Path to the folder containing the 6 cubemap images
     var srcPath = data.folder;
+    if (srcPath === '')
```

Comment:
Style-wise I prefer to always use brackets in `if` statements (even if they're not strictly necessary).